Spokestack/AppleSpeechRecognizer.swift

```diff
@@ -178,7 +187,9 @@ extension AppleSpeechRecognizer: SpeechProcessor {
             self.recognitionRequest = nil
         }
     }
-    
+
+    /// Processes an audio frame, detecting speech.
```

Comment:
This is a tad confusable with the VAD documentation—maybe "transcribing" or "attempting to recognize"?